gradleResources/staticResources/js/bsf.js

```diff
@@ -110,7 +110,10 @@ BsF.ajax.callAjax = function(source, event, update, execute, oncomplete,
 		}
 	}
 	jsf.ajax.request(source, event, opts);
-	if ($.blockUI && $.blockUI != null) {
+	
+	var disableBlockUI = opts['blockui.disabled'] === "true" ? true : false;
+	
```

Comment:
IMHO you can safely omit the elvis operator. A simple 
```
var disableBlockUI = opts['blockui.disabled'] === "true";
```
does the trick, too :).